### PR TITLE
Include full copy of Apache 2.0 license

### PR DESCRIPTION
**Pull request by sverhoeven**
GH does not recognize the notice as Apache 2.0

### CODE REVIEW

**Comment by sverhoeven** [comment]
This will get us a silver badge on https://sqaaas.eosc-synergy.eu/

File: LICENSE [hunk 1]
```diff
@@ -1,16 +1,201 @@
...
+      To apply the Apache License to your work, attach the following
+      boilerplate notice, with the fields enclosed by brackets "[]"
+      replaced with your own identifying information. (Don't include
+      the brackets!)  The text should be enclosed in the appropriate
+      comment syntax for the file format. We also recommend that a
+      file or class name and description of purpose be included on the
+      same "printed page" as the copyright notice for easier
+      identification within third-party archives.
+
+   Copyright [yyyy] [name of copyright owner]
```
Comment (BSchilperoort) [inline LICENSE:189]: Would we want to enter something here instead of the placeholders?

**Review by BSchilperoort — APPROVED** [review]
Nice! I didn't see any issues (other than the placeholder year/name).
We will get the silver badge now 🥈 

Although it seems that for gold we'll need to set up a (flake8) linter (and fix the issues), so I guess we'll settle for silver.

File: LICENSE (hunk 1 above)
Comment (sverhoeven) [reply LICENSE:189]: The copyright headers are optional so I think the placeholders are as well.

**Comment by sverhoeven** [comment]
Thanks for reviewing.

We have flake8 warnings in [pre-commit](https://github.com/eWaterCycle/ewatercycle/blob/main/.pre-commit-config.yaml#L35), but solving them is a task we have been pushing into the future for a while now.

**Pull request merged**